src/components/CourseForm/CourseForm.jsx

```diff
@@ -51,32 +53,68 @@ const InputAuthor = styled.input`
 	width: 100%;
 `;
 
-const CreateCourse = ({
+const CourseForm = ({
 	authorsForm,
 	setAuthors,
 	addAuthorToForm,
 	deleteAuthor,
 	resetForm,
 	enabledAuthors,
+	resetAuthorForm,
 }) => {
 	const [authorInput, setAuthorInput] = useState('');
 	const [duration, setDuration] = useState('');
 	const [title, setTitle] = useState('');
 	const [descr, setDescr] = useState('');
 	const history = useHistory();
 	const token = useSelector((state) => state.authentication.token);
+	const dispatch = useDispatch();
+	const { courseId } = useParams();
+	const courseStore = useSelector((state) => state.courses.courses);
```

Comment:
I'd recommend to retrieve only single course info using endpoint /courses/:id
In this case you won't need updatedCourse state